src/Querying/QueryDescription.php

```diff
@@ -87,6 +87,11 @@ public function returnType(Type $typeSchema): self
         return $this;
     }
 
+    public function getReturnType(): ?array
```

Comment:
just a minor change request:

Let's:  
- remove the method `queryComplexity` (bc related to GraphQL but GraphQL will be moved to optional package)
- rename `returnType` -> `setReturnType`
- renmae `getReturnType` -> `returnType`

Reply:
I can definitely make the changes to the function names. It will be a backwards incompatible change to the publicly used API of the class, though. That is fine since the project is still pre-release, but we will need to make sure we update documentation/sample projects.

Reply:
good catch! But let's do it. Event Machine is only used in small sample apps yet so breaking the public API is ok. Tutorial needs to be updated, though